optimade/server/tests/test_more_structures.json

```diff
@@ -1,30 +1,30 @@
-{
-  "id": "db/1234567",
+[{
+  "task_id": "db/1234567",
   "type": "structure",
 	"last_modified": {
 		"$date": "2019-06-08T05:13:37.331Z"
 	},
-  "_exmpl_band_gap": 1.23456,
+  "band_gap": 1.23456,
+	"_mp_chemsys": "Ge-Si",
   "elements": ["Si", "Ge"],
-  "nelements": 3,
+  "nelements": 2,
+  "elements_ratios": [0.5, 0.5],
+  "pretty_formula": "SiGe",
+  "formula_anonymous": "GeSi",
```

Comment:
This should now be "AB" according to https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#627chemical_formula_anonymous

Reply:
Ah, sure. I just put in some junk :) But yeah, you're right.